sdk/schemaregistry/schema-registry/src/conversions.ts

```diff
@@ -53,14 +59,15 @@ export function convertSchemaIdResponse(response: GeneratedSchemaIdResponse): Sc
 /**
  * Converts common portion of all generated client responses.
  */
-function convertResponse(response: GeneratedResponse): SchemaIdResponse {
+function convertResponse(response: GeneratedResponse): SchemaId & Response {
   return {
     _response: response._response,
```

Comment:
the downside here is _response is now an enumerable property, whereas we usually make it non-enumerable so folks don't accidentally try to print/JSON.serialize it. Could you switch this to using `Object.defineProperty` to set _response instead?

Reply:
Thanks. I just learned about this distinction now. Will fix. 😃